config/checkstyle/checkstyle.xml

```diff
@@ -46,9 +46,14 @@ page at http://checkstyle.sourceforge.net/config.html -->
             <!-- Put a comment in it if you need one explaining why -->
             <property name="option" value="text" />
         </module>
-        <module name="LeftCurly">
+        <!-- This is preventing us from being able to keep small functions on one line:
```

Comment:
I'm not really a fan of one line functions, but I know it is a matter of taste.

Reply:
I wanted to give it a chance, but now I feel similarly. There is a feature in IntelliJ that displays short functions over multiple lines as if they were 1 line functions. I think if we configure this then we can uncomment this rule but still be able to view short functions however we like in our editor. We'll discuss when I get back.